src/data/markdown/docs/02 javascript api/06 k6-execution.md

```diff
@@ -74,6 +74,13 @@ Meta information and execution details about the current vu and iteration.
 | iterationInScenario | integer | The identifier of the iteration in the current scenario. |
 | idInInstance        | integer | The identifier of the VU across the instance. |
 | idInTest            | integer | The globally unique (across the whole test run) identifier of the VU. |
+| tags                | object  | The map that gives control over VU's Tags. The Tags will be included in every metric emitted by the VU and the Tags' state is maintained across different iterations while the VU exists. |
```

Comment:
hm ... doesn't this reset between scenarios though :thinking:

Reply:
Same here, I thought it was gonna reset as the cookies do.

Reply:
it seems so from thsi code https://github.com/grafana/k6/blob/master/js/runner.go#L606-L623